python/paddle/device/cuda/__init__.py

```diff
@@ -204,3 +199,63 @@ def stream_guard(stream):
             yield
         finally:
             stream = _set_current_stream(pre_stream)
+
+
+def get_device_name(device=None):
+    '''
+    Return the name of the device which is get from CUDA function `cudaDeviceProp<https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__DEVICE.html#group__CUDART__DEVICE_1g1bf9d625a931d657e08db2b4391170f0>`_..
+
+    Parameters:
+        device(paddle.CUDAPlace()|int, optional): The device or the ID of the device.
```

Comment:
The `device` can be a string like `gpu:xxx`. Same below.

Reply:
The input device only support int and paddle.CUDAPlace() which is the same with pytorch implement.